src/components/RouteOption/RouteOption.jsx

```diff
@@ -1,17 +1,45 @@
+// import React from "react";
+// import PropTypes from "prop-types";
+// import DockedEbikeRouteOption from "./DockedEbikeRouteOption";
+
+// export default function RouteOption({type, ...props}) {
+//   if (type === DockedEbikeRouteOption.TYPE) {
+//     return DockedEbikeRouteOption(props);
+//   } else {
+//     throw TypeError(`Unknown route option type: '${type}'.`);
+//   }
+// }
+
+// RouteOption.propTypes = {
+//   type: PropTypes.oneOf([
+//     DockedEbikeRouteOption.TYPE,
+//   ]),
+// }
+
+
 import React from "react";
 import PropTypes from "prop-types";
 import DockedEbikeRouteOption from "./DockedEbikeRouteOption";
 
-export default function RouteOption({type, ...props}) {
+export default function RouteOption({ type, isSelected, onClick, ...props }) {
+  const style = {
+    backgroundColor: isSelected ? "blue" : "transparent",
+    cursor: "pointer",
+  };
+
   if (type === DockedEbikeRouteOption.TYPE) {
-    return DockedEbikeRouteOption(props);
+    return (
+      <div onClick={onClick} style={style}>
+        <DockedEbikeRouteOption {...props} />
+      </div>
+    );
   } else {
-    throw TypeError(`Unknown route option type: '${type}'.`);
+    throw new TypeError(`Unknown route option type: '${type}'.`);
   }
 }
 
 RouteOption.propTypes = {
-  type: PropTypes.oneOf([
-    DockedEbikeRouteOption.TYPE,
-  ]),
-}
+  type: PropTypes.oneOf([DockedEbikeRouteOption.TYPE]),
+  isSelected: PropTypes.bool,
+  onClick: PropTypes.func,
```

Comment:
if we move this to `DockedEbikeRouteOption`, please make both properties mandatory, i.e.,

```javascript
isSelected: PropTypes.bool.isRequired,
onClick: PropTypes.func.isRequired,
```

Reply:
done in https://github.com/Micro-Commute/micro-commute-ui/pull/18/commits/36dd2b9f13194bc7f212c043ffda2d595ea8e6ea and https://github.com/Micro-Commute/micro-commute-ui/pull/18/commits/b99181b0c5595a341c20b056dcdf648bee0d9fde